### PR TITLE
Display message after clicking Save

### PR DESCRIPTION
**Pull request by awead**
Friday. Begins with F, and that stands for **Fancy**!

Here's a video of the interaction with the screen reader voice over. You'll need to turn your volume up to 11 to hear it.

https://www.dropbox.com/s/u2858kwnplbv9a0/demo.mov?dl=0

Might need some css tweaks. @mtribone @cam156

### CODE REVIEW

File: app/assets/javascripts/scholarsphere_fileupload.js [hunk 1]
```diff
@@ -1,15 +0,0 @@
-//500 MB  max file size 
-max_file_size = 500000000;
```
Comment (carolyncole) [inline app/assets/javascripts/scholarsphere_fileupload.js:2]: @awead why did you remove the maximums?  These override items in curation concerns https://github.com/projecthydra/curation_concerns/blob/master/app/assets/javascripts/curation_concerns/uploader.js#L13

File: app/views/curation_concerns/base/_form_progress.html.erb [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<aside id="form-progress" class="form-progress panel panel-default">
```
Comment (carolyncole) [inline app/views/curation_concerns/base/_form_progress.html.erb:1]: Can you add a comment to the top of the class to describe why we are overriding?

**Review by carolyncole — CHANGES_REQUESTED** [review]
@awead looks good to me.  I like that it speaks!  Just a couple little tweaks.

File: app/assets/javascripts/scholarsphere_fileupload.js (hunk 1 above)
Comment (awead) [reply app/assets/javascripts/scholarsphere_fileupload.js:2]: @cam156 these don't do anything anymore. The size restriction is now in:
https://github.com/psu-stewardship/scholarsphere/blob/develop/app/assets/javascripts/sufia/uploader.js#L34-L35

File: app/views/curation_concerns/base/_form_progress.html.erb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    <br>
+    <%= link_to t(:'helpers.action.cancel'),
+                sufia.dashboard_index_path,
+                class: 'btn btn-default' %>
+    <%= f.submit 'Save', class: 'btn btn-primary', onclick: "confirmation_needed = false;", id: "with_files_submit", name: "save_with_files" %>
+
+  </div>
+  <div class="panel-footer hidden">
+    <div class="progress">
+      <div class="progress-bar progress-bar-striped active" style="width: 100%">
```
Comment (mtribone) [inline app/views/curation_concerns/base/_form_progress.html.erb]: Can the inline style for the width be applied to a class that you are using?

File: app/views/curation_concerns/base/_form_progress.html.erb (hunk 2 above)
Comment (awead) [reply app/views/curation_concerns/base/_form_progress.html.erb:1]: @cam156 I put the comment above the spot at which we're overriding.

File: app/views/curation_concerns/base/_form_progress.html.erb (hunk 3 above)
Comment (awead) [reply app/views/curation_concerns/base/_form_progress.html.erb]: @mtribone done.

**Review by carolyncole — APPROVED** [review]

**Pull request merged**